src/transformers/models/luke/modeling_luke.py

```diff
@@ -674,6 +717,38 @@ def forward(self, hidden_states):
         return pooled_output
 
 
+class EntityPredictionHeadTransform(nn.Module):
```

Comment:
This could benefit from a `# Copied from` statement originating from others, like BERT!